pkg/apis/workspace/workspace.go

```diff
@@ -0,0 +1,19 @@
+package workspace
+
+// Workspace is a logical concept representing a target that stacks will be deployed to.
+// Workspace is managed by platform engineers, which contains a set of configurations
+// that application developers do not want or should not concern, and is reused by multiple
+// stacks belonging to different projects.
+type Workspace struct {
+	// Name identifies a Workspace uniquely.
+	Name string `yaml:"-" json:"-"`
```

Comment:
Why should we ignore the `name` field in yaml and json, could you provide additional comments to clarify this?

Reply:
This is because there is no `name` field in the configuration file